api/tests/shared/acceptance/application/swagger_test.js

```diff
@@ -1,11 +1,13 @@
 import { createServer, expect } from '../../../test-helper.js';
 
 describe('Acceptance | Controller | swagger', function () {
+  // Increase the test timeout because swagger.json endpoints can be long to generate/respond.
+  this.timeout(5000);
+
   let server;
 
   beforeEach(async function () {
     server = await createServer();
-    this.timeout(3000);
```

Comment:
**Remarque:** Mettre le timeout dans un `beforeEach` n'a aucun effet, il faut soit le mettre dans un `describe` ou un `it`

Reply:
Il n'y a pas de règle de lint qui pourrait bloquer cet usage ? ^^"